packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -79,36 +83,50 @@ fi
 if [[ -z $DOCKER_IMAGE_NAME ]]; then
   if [[ $BUILD_LAYER_WITH == local ]]; then
     DOCKER_IMAGE_NAME=instana-aws-lambda-nodejs-local
+  elif [[ $BUILD_LAYER_WITH == npm ]] && [[ $LAYER_NAME == instana-nodejs ]]; then
+      DOCKER_IMAGE_NAME=$CONTAINER_REGISTRY/instana/aws-lambda-nodejs
   else
-    DOCKER_IMAGE_NAME=$CONTAINER_REGISTRY/instana/aws-lambda-nodejs
+    DOCKER_IMAGE_NAME=$CONTAINER_REGISTRY/instana/aws-lambda-nodejs-experimental
```

Comment:
This prevents deploying a docker image with the name "/instana/aws-lambda-nodejs" when using an experimental layer name